packages/e2e/package.json

```diff
@@ -42,18 +42,22 @@
     "@accounts/server": "^0.9.3",
     "@accounts/typeorm": "^0.9.3",
     "@accounts/types": "^0.9.3",
-    "@graphql-modules/core": "0.2.18",
+    "@graphql-modules/core": "0.4.2",
     "@types/lodash": "4.14.120",
     "@types/mongoose": "5.3.10",
     "@types/node-fetch": "2.1.4",
     "apollo-boost": "0.1.27",
-    "apollo-server": "2.3.1",
+    "apollo-server": "2.3.3",
+    "core-js": "3.0.0-beta.3",
     "body-parser": "1.18.3",
     "express": "4.16.4",
-    "graphql": "14.0.2",
+    "graphql": "14.1.1",
     "lodash": "4.17.11",
     "mongoose": "5.4.7",
     "node-fetch": "2.3.0",
     "typeorm": "0.2.11"
+  },
+  "resolutions": { 
+    "core-js": "3.0.0-beta.11" 
```

Comment:
Unfortunately resolution does not work on subpackages with workspaces so I think you can remove it :)